couchbase-edge-server/metadata/allowed-metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "note": "Please update the Wiki page https://hub.internal.couchbase.com/confluence/display/CR/Supported+Metadata+Entries+for+Build+Database for any changes",
```

Comment:
Should update this URL to the new Wiki. (should probably make a separate change to fix that in all the other products too.)

Reply:
updated URLs for all.